### DISCUSSION TITLE
Finding empty translations

### DISCUSSION THREAD

**Discussion by JorgenSolli** [MohmmedAshraf/laravel-translations#38] [Q&A] [answered]
Hey! Thanks for this package. By the looks of it, this can save me a lot of time :)

I do have one question tho. Hopefully, I'm just not seeing it.
When I add a new translation in English, the other languages will now have a key without a translation. How do I quickly find these entries without searching for the key specifically?

I might not be translating all languages, so it would be nice for someone else to quickly jump into their language and sort by keys not yet translated. It looks like the table is sorted alphabetically now, so any new translations added will be hard to find.

**Comment by JorgenSolli** [comment]
Pardon me. I only searched in Issues, so I did not realize this was a [duplicate](https://github.com/MohmmedAshraf/laravel-translations/discussions/35)

**Comment by MohmmedAshraf** [comment, marked as answer]
Hey @JorgenSolli! First of all, I want to apologize for the delay in my response. I've been quite busy lately. It's great to hear that the package has been helpful to you.

Regarding your question, I have exciting plans for the package, including the implementation of a feature you mentioned. I'll be actively working on it starting next week. If you have any additional questions or suggestions, please don't hesitate to share them with me. I'm here to help!
